packages/cloud2edge/requirements.yaml

```diff
@@ -12,8 +12,8 @@
 #
 dependencies:
   - name: hono
-    version: ^1.10.18
+    version: ~2.0.7
     repository: "https://eclipse.org/packages/charts/"
   - name: ditto
-    version: ~2.4.0
+    version: ~2.4.1
```

Comment:
@kalinkostashki this must be the chart version of Ditto (you used the "appVersion") - the most recent chart version is 2.5.5 (including Ditto 2.4.1)

Reply:
done